examples/graphics/surface.cpp

```diff
@@ -13,8 +13,8 @@
 
 using namespace af;
 
-static const int ITERATIONS = 30;
-static const float PRECISION = 1.0f/ITERATIONS;
+static const int POINTS = 30;
```

Comment:
POINTS is a typedef to a struct on Windows in windef.h. That's causing build failure.